youtube_dl/options.py

```diff
@@ -709,6 +709,10 @@ def _comma_separated_values_options_callback(option, opt_str, value, parser):
         '-A', '--auto-number',
         action='store_true', dest='autonumber', default=False,
         help=optparse.SUPPRESS_HELP)
+    filesystem.add_option(
+        '--autonumber-reset',
+        action='store_true', dest='autonumber_reset', default=False,
+        help='Reset autonumber counter to zero before each playlist')
```

Comment:
`autonumber` is not reset to zero in the first place.

Reply:
Updated documentation in options and YoutubeDL to reflect that it is reset to the current `--autonumber-start` value, not zero